RELEASE_NOTES.md

```diff
@@ -55,6 +55,10 @@ For more details and a list of all deprecated CSM features, see [Deprecations](i
 
 ## Removals
 
+* The [Content Projection Service (CPS)](glossary.md#content-projection-service-cps) and the
+  [Data Virtualization Service (DVS)](glossary.md#data-virtualization-service-dvs)
+    * These are replaced by the [Scalable Boot Projection Service](../../glossary.md#scalable-boot-projection-service-sbps)
```

Comment:
Technically SBPS replaces DVS for image projection, but DVS still remains available for the projection of customer file systems, but only on dedicated gateway nodes.   Not sure if we need to include that caveat.

Reply:
```suggestion
* Support for projecting root filesystems and PE images using the [Content Projection Service (CPS)](glossary.md#content-projection-service-cps) and the
  [Data Virtualization Service (DVS)](glossary.md#data-virtualization-service-dvs)
    * This projection is now done using the [Scalable Boot Projection Service](../../glossary.md#scalable-boot-projection-service-sbps)
```